Python/Tests/DebuggerUITestsRunner/DebugProjectUITests.cs

```diff
@@ -266,7 +266,7 @@ public class DebugProjectUITestsExperimental27 : DebugProjectUITestsExperimental
     }
 
     [TestClass]
-    public class DebugProjectUITestsExperimental36 : DebugProjectUITestsExperimental {
-        protected override string Interpreter => "Python36|Python36_x64";
+    public class DebugProjectUITestsExperimental37 : DebugProjectUITestsExperimental {
+        protected override string Interpreter => "Python37|Python37_x64";
```

Comment:
is this for the last version of Python 3.x? I wonder.

Reply:
Oh, I noticed that and forgot to mention anything. I'm pretty sure this one has to match one of the test projects under TestData. If so, you'll have to update those to use Python 3.7 as well.

Reply:
@zooba, do you happen to know which one? I can't find it and most of the tests are passing under 37.

Reply:
@zooba, do you happen to know which one? I can't find it and most of the tests are passing under 37.

Reply:
Looking through the referenced project files, it looks like `TestData\SysPath\SysPath.pyproj` and `DebugAttach\DebugAttach.pyproj` expect to use Python 2.7, but all the rest are going to use whatever is the active default. So this is probably fine.